runner/src/server/services/uploadService.ts

```diff
@@ -224,4 +247,31 @@ export class UploadService {
     const promises = paths.map((path) => get<string>(path, {}));
     return Promise.all(promises);
   }
+
+  async uploadFilesS3(files, filePrefix) {
+    let response = new Array();
+
+    for (const file of files) {
+      const uploadParams = {
+        Bucket: bucketName,
+        Body: file,
+        Key: `${filePrefix}/${file.hapi.filename}`,
```

Comment:
Can we pass through the Content Type here as well?

Reply:
added the content type